tests/test_space.py

```diff
@@ -13,6 +14,21 @@ def test_space_simple1(adata):
     space(adata, marker="*", save_show_or_return="show")
 
 
+def test_space_stack_color(adata):
+    adata = read_test_spatial_genomics_data()
+    genes = adata.var_names[:18]
+    # space(adata, genes=genes, marker="*", save_show_or_return="show", stack_genes=False)
+    space(
+        adata,
+        genes=genes,
+        marker=".",
+        save_show_or_return="save",
+        stack_genes=True,
+        alpha=0.5,
+        show_colorbar=False,
```

Comment:
can we set show_colorbar to be True here?

Reply:
(I mean, set it True in the pl.space function )

Reply:
No for now. Setting color bar makes the whole plot messy because it is not easy to handle the colorbar parts in _matplotlib_points and scatters.py ~1000 lines plotting funciton.

Reply:
ok, we can ignore this for now